src/renderer/SideBar.tsx

```diff
@@ -16,7 +16,7 @@ const SideBar = () => {
 
   const setGain = async (newValue: number) => {
     try {
-      await setAutoPreAmpOn(false);
+      setAutoPreAmpOn(false);
       await setMainPreAmp(newValue);
```

Comment:
We forgot to tell backend to update autopreamp setting when we first wrote this file. lets fix this.

Reply:
wdym by this?

Reply:
The expected behavior is that we want to save the state of `isAutoPreAmpOn` in the BE whenever we change it. This is an instance of us changing the state in the UI but not updating the BE. We need to add:

```

Reply:
added